packages/react-api/src/api/SQL.js

```diff
@@ -41,16 +50,23 @@ export const executeSQL = async (credentials, query, opts = { format: '' }) => {
  * Create an 'SQL query' request
  * (using GET or POST request, depending on url size)
  */
-function createRequest({ credentials, query, opts }) {
+function createRequest({ credentials, connection, query, opts = {} }) {
   const { abortController, ...otherOptions } = opts;
 
+  const { apiVersion = API_VERSIONS.V2 } = credentials;
```

Comment:
nice destructuring with default

Reply:
I'm getting a master in javascript from the best people :)